must.d.ts

```diff
@@ -73,6 +73,7 @@ interface CallableMust extends Must {
     (): Must;
 }
 declare function must(expected: any): Must;
```

Comment:
I think you can safely remove this line as `msg` is optional

Reply:
You are correct Chris, just left it there for explicitness reasons but it can safely be removed.